### ISSUE TITLE
SIA storage

### ISSUE THREAD

**Issue by alyasboytigas** [HandyOSS/HandyHost#30]
For some odd reason the save button seems not working. I tried adding storage and save but it seems there's no loading or it's not saving. I tried restart the handyhost and pc no luck. tried navigating away still the same. any suggestions? thanks.

**Comment by timrettop** [comment]
I had the same issue and tracked it down to the folder not being writable. In my case it was due to nfs and nosquashroot but I guess it could be a lot of things.

**Comment by alyasboytigas** [comment]
thanks, I'll check the permission.

**Comment by alyasboytigas** [comment]
permission seems fine... maybe it takes a while I mean my SIA been running couple weeks its just today I restarted the pc. weird tho.

**Comment by alyasboytigas** [comment]
i formatted the storage USB drive and still no good.

**Comment by alyasboytigas** [comment]
Here's what I did:

1. Format the drive
2. sudo fdisk -l
3. mkdir /media/usb-drive2
4. mount /dev/sdc2 /media/usb-drive2/
5. mount | grep sdc1 (Check if it mounted)
6. cd /media/usb-drive2 (check if I can access it which i can)
7. ls and add test folder successful.

I don't know what else I'm missing when I first started configuring the storage on SIA this is what I did. Just a simple restart on PI seems can't add on the SIA browser.

Any helps I appreciate it. Thanks.
